README.md

```diff
@@ -9,3 +9,5 @@
 ### PS
 IN TRAINING : Due to communication error when running the valid dataset on a single main GPU, a multi machine and multi GPUS solution is temporarily used to run all train and valid datasets
 ATTENTION : Please ensure that the torch family bucket you need matches your CUDA version
+
+To install, first create a conda environment from environment.yaml and then run setup.sh.
```

Comment:
I think it might be more obvious if that is put in a requirements file, like `requirements-pyg.txt` and then referenced here with `pip install -r ...`. At least then you know you need to activate the requirement first and it's not so opaque what's going on.